config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 require 'rails/all'
+require 'elasticsearch/rails/instrumentation'
```

Comment:
For whatever reason, it really only works in the `application.rb` file, not in an initializer. If you find out why and how to potentially fix it, I owe you :)